src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java

```diff
@@ -59,17 +63,18 @@ public UnmarkAttendanceByStudentCommand(Index targetIndex, Attendance attendance
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
+        logger.info("Running execute(Model model)");
         List<Person> lastShownList = model.getFilteredPersonList();
 
         if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            logger.warning(String.format(MESSAGE_LOGGER_FOR_EXCEPTION, UnmarkAttendanceByStudentCommand.class
```

Comment:
Good use of logger to inform about exceptions thrown, so users will know which part of the code is causing the error